packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -229,7 +229,12 @@ const Router = Backbone.Router.extend({
         uniqueUserId,
       } = this.metrics.getFilteredData();
 
-      const settingsLink = `/settings${Url.objToSearchString({
+      // Our GQL server sets the `redirect_to` param if a user attempts
```

Comment:
I could be missing it but I think we're talking about the `redirect_to` in [this file ](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/gql.ts#L31) right? Might be more appropriate for the comment to reference the GQL client rather than the server.